cla_frontend/apps/status/smoketests/basic.py

```diff
@@ -17,26 +17,27 @@
 @live_smoketests.register(1, "Public site is up")
 @ready_smoketests.register(1, "Public site is up")
 def things_exist():
-    response = get(fe("/"))
+    response = get_fe("/auth/login/")
     assert_status(response, 200)
 
 
-@live_smoketests.register(2, "Angular is loaded")
-@ready_smoketests.register(2, "Angular is loaded")
+# Skipping until LGA-1612 (s3 static storage)
+# @live_smoketests.register(2, "Angular is loaded")
```

Comment:
Was this disable because of the issue we fixed of npm install and gulp being run in the wrong directory? If so can we re-enable again?

Reply:
It was disabled until the static files are set to be served by s3 (as it would fail with DEBUG=False with local storage)
So we should re-enable it in that PR, good catch.

But this one is fine, as this PR won't serve them.